lib/pbench/__init__.py

```diff
@@ -17,7 +17,7 @@
 _STD_DATETIME_FMT = "%Y-%m-%dT%H:%M:%S.%f"
 
 
-class PbenchConfig(object):
+class PbenchConfig:
```

Comment:
See https://github.com/distributed-system-analysis/pbench/pull/1692/commits/467bf57f41d16e553c7289fb7f4e1fac0d4b3e9c.  There was a review comment asking to remove the addition of `(object)` to class definitions since in Python3 you no longer need to sub-class `object` explicitly.